modules/apps/transfer/keeper/relay_forwarding_test.go

```diff
@@ -1,15 +1,13 @@
 package keeper_test
```

Comment:
The changes on this file seem to mix the purpose of the PR (use registered error code) with some other refactoring. WDYT about moving them into a separate PR?

Reply:
It does to some extent, but a good chunk of the refactor was to make use of the added error code in a test. Some of it I could've potentially skipped, but it would only be a relatively small portion. I did expect the refactor to be slightly smaller, though. If there are concerns about the refactor itself, I could take it out and not test that error message. Wdyt?

Reply:
It's fine by me to let it like this. But since my understanding of the test code is still fuzzy, I'll just refrain from LGTMing it :)

Reply:
In general, less diffs are always appreciated. But this felt like an okay amount.